### PR TITLE
add Ed25519 ciphersuite

### LINKED ISSUE #70: Add Ed25519 support
We want to support all curves defined in the spec.

### PR DESCRIPTION
**Pull request by conradoplg**
Based on #114 

Closes #70

### CODE REVIEW

File: frost-ed25519/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[package]
+name = "frost-ed25519"
+edition = "2021"
+# When releasing to crates.io:
+# - Update html_root_url
+# - Update CHANGELOG.md
+# - Create git tag.
+version = "0.1.0"
+authors = ["Deirdre Connolly <durumcrustulum@gmail.com>", "Chelsea Komlo <me@chelseakomlo.com>"]
```
Comment (dconnolly) [inline frost-ed25519/Cargo.toml:9]: Aren't you a co-author, @conradoplg ? :D

File: frost-ed25519/src/tests.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+lazy_static! {
+    pub static ref ED25519_SHA512: Value =
+        serde_json::from_str(include_str!("tests/vectors.json").trim())
+            .expect("Test vector is valid JSON");
+}
+
+/// This is testing that Shamir's secret sharing to compute and arbitrary
+/// value is working.
+#[test]
+fn check_share_generation_ristretto255_sha512() {
```
Comment (dconnolly) [inline frost-ed25519/src/tests.rs:16]:
```suggestion
fn check_share_generation_ed25519_sha512() {
```

**Review by dconnolly — APPROVED** [review]
🎉

File: frost-ed25519/Cargo.toml (hunk 1 above)
Comment (conradoplg) [reply frost-ed25519/Cargo.toml]: I guess I am :laughing: Added in 5687e4b25860e

**Review by dconnolly — APPROVED** [review]

**Pull request merged**